src/components/EventsDisplay/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import { Card, Button, Row, Col } from "react-bootstrap";
+import {
+  Element,
+  animateScroll,
+} from "react-scroll";
+
+import styles from "./styles.module.css";
+
+export default class EventsDisplay extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      events: props.events,
+    };
+  }
+
+    render() {
+        return (
+          <Element
+            className="element"
+            id="scroll-container"
+            style={{
+              height: "270px",
```

Comment:
Move this over to the `styles.module.css` file!

Reply:
RESOLVED: done!